.github/workflows/docker-build.yml

```
@@ -0,0 +1,125 @@
+name: docker-build
```

actionlint (1.7.4)

"on" section is missing in workflow

(syntax-check)